File: C_PROGRAMMING/C_PROGRAMS/Statements/0_Basics/program7.c

// WAP  take a number and print whether it is less than 10 or greater than 10, take all values from users....
//
#include<stdio.h>
void main(){
  int num;
  printf("Enter the number you want to check:");
  scanf("%d",&num);

  if(num > 10){
    printf("\n %d is Greater than 10\n",num);
   }
  else
  {
    printf("\n %d is less than 10\n",num);

  }
}

File: JAVA_PROGRAMMING/ASSIGNMENTS/CONTROL_STATEMENTS/Pattern/Assignment_1/program7.java

// WAP to print following pattern
//
//	1    2    9
//	4    25   6
//	49   8    81


class pattern{

	public static void main(String[] args){
		int num = 1;

		for(int i = 1; i <= 3;i++){

			for(int j = 1; j <= 3; j++){
				if(num % 2== 1)
					System.out.print((num*num) +"\t");
				else
					System.out.print(num+ "\t");

				num++;
			}
		
		System.out.println();
		}
	}
}
